src/tokens/TokenImport.test.tsx

```diff
@@ -24,6 +60,28 @@ describe('TokenImport', () => {
     expect(getByText('tokenImport.importButton')).toBeTruthy()
   })
 
+  it('enables the import button when form is filled', async () => {
+    const store = createMockStore({})
+    const { getByText, getByPlaceholderText, getByTestId } = render(
+      <Provider store={store}>
+        <TokenImportScreen {...mockScreenProps} />
+      </Provider>
+    )
+    const tokenAddressInput = getByPlaceholderText('tokenImport.input.tokenAddressPlaceholder')
+    const tokenSymbolInput = getByTestId('tokenSymbol')
+    const importButton = getByText('tokenImport.importButton')
+
+    fireEvent.changeText(tokenAddressInput, mockPoofAddress)
+    expect(tokenSymbolInput).toBeDisabled()
+    expect(importButton).toBeDisabled()
+    fireEvent(tokenAddressInput, 'blur')
+
+    await waitFor(() => {
+      expect(tokenSymbolInput.props.value).toBe('ABC')
```

Comment:
nit: it'd be nice to have only 1 thing inside the waitFor https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#having-multiple-assertions-in-a-single-waitfor-callback